.travis.yml

```diff
@@ -1,20 +1,46 @@
 sudo: no
 language: python
-env:
-  matrix:
-    - TOXENV=docs
-    - TOXENV=py27-1.6
-    - TOXENV=py27-1.7
-  global:
-    - PIP_WHEEL_DIR=$HOME/.wheels
-    - PIP_FIND_LINKS=file://$PIP_WHEEL_DIR
-cache:
-  directories:
-    - $PIP_WHEEL_DIR
+cache: pip
+matrix:
+  include:
+    - python: "2.7"
```

Comment:
I see you're using generated envs in tox. Excellent. You might be able to save some duplication here by doing something like https://github.com/erikrose/peep/blob/master/.travis.yml#L23.